fiona/collection.py

```diff
@@ -427,6 +430,30 @@ def __del__(self):
         self.close()
 
 
+def _get_valid_geom_types(schema, driver):
+    schema_geom_type = schema["geometry"]
+    if isinstance(schema_geom_type, string_types) or schema_geom_type is None:
+        schema_geom_type = (schema_geom_type,)
+    valid_types = set()
+    for geom_type in schema_geom_type:
+        valid_types.add(str(geom_type).lstrip("3D "))
+
+    if "Unknown" in valid_types:
+        ALL_GEOMETRY_TYPES = set([
+            geom_type for geom_type in GEOMETRY_TYPES.values()
+            if "3D " not in geom_type and geom_type != "None"])
+        valid_types.update(ALL_GEOMETRY_TYPES)
+        valid_types.remove("Unknown")
+
+    if driver == "ESRI Shapefile" and "Point" not in valid_types:
+        print("OMG")
```

Comment:
Funny, but I think we should remove this :)

Reply:
Ha! This wasn't supposed to be left in. It's my go-to statement for checking branches are being run. I'll remove it.